onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -440,6 +440,9 @@ static void LogNodeSupport(const logging::Logger& logger,
   logging::Capture(logger, log_severity, logging::Category::onnxruntime,
                    log_data_type, call_site)
           .Stream()
+      << (support_status.IsOK() ? "Validation PASSED " : "Validation FAILED ") << "for " << num_nodes
+      << " nodes in " << qnn_node_group.Type() << " (" << qnn_node_group.GetTargetNodeUnit()->OpType() << ") :"
+      << std::endl
```

Comment:
Note: These changes log the individual nodes supported within a NodeUnit or fusion (e.g., ConvReluFusio):
```
[... onnxruntime::QNNExecutionProvider::GetSupportedNodes] Validation PASSED for 3 nodes in NodeUnit (Unsqueeze) :
	Operator type: DequantizeLinear Node name: /model/embed_tokens/Gather_output_0_dq Node index: 6063
	Operator type: Unsqueeze Node name: /model/embed_tokens/Gather_output_0_Unsqueeze Node index: 198
	Operator type: QuantizeLinear Node name: /model/embed_tokens/Gather_output_0_unsqueeze_QuantizeLinear Node index: 200
```